packages/compiler/src/render3/view/template.ts

```diff
@@ -1024,21 +1031,23 @@ export class TemplateDefinitionBuilder implements t.Visitor<void>, LocalResolver
   private bindingContext() { return `${this._bindingContext++}`; }
 
   private templatePropertyBindings(
-      template: t.Template, templateIndex: number, attrs: (t.BoundAttribute|t.TextAttribute)[]) {
+      templateIndex: number, attrs: (t.BoundAttribute|t.TextAttribute)[]) {
+    const propertyBindings: ChainablePropertyBinding[] = [];
     attrs.forEach(input => {
       if (input instanceof t.BoundAttribute) {
         const value = input.value.visit(this._valueConverter);
 
         if (value !== undefined) {
           this.allocateBindingSlots(value);
-          this.updateInstruction(
-              templateIndex, template.sourceSpan, R3.property,
-              () =>
-                  [o.literal(input.name),
-                   this.convertPropertyBinding(value, /* skipBindFn */ true)]);
+          propertyBindings.push(
+              {name: input.name, input, value: () => this.convertPropertyBinding(value, true)});
```

Comment:
Nit: Looks like we lost the inline comment here. Re-add?

```suggestion
              {name: input.name, input, value: () => this.convertPropertyBinding(value, /* skipBindFn */ true)});
```